### PR TITLE
fixes #27230 - SysPurpose act key info

### PR DESCRIPTION
**Pull request by sseelam2**

### CODE REVIEW

File: test/functional/activation_key/info_test.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+require_relative '../test_helper'
+require 'hammer_cli_katello/associating_commands'
+
+describe 'activation-key info' do
+  before do
+    @cmd = %w(activation-key info)
+  end
+
+  it "includes katello attributes" do
+    params = ['--id=1', '--organization-id=1']
```
Comment (akofink) [inline test/functional/activation_key/info_test.rb]: No need for `--organization-id` if you are using `--id`

File: test/functional/activation_key/update_test.rb [hunk 2]
```diff
@@ -19,7 +19,7 @@ module HammerCLIKatello
           p['id'] == 6 && p['organization_id'] == 1
         end
 
-        run_cmd(%w(activation-key update --organization org1 --name ak --auto-attach false))
+        run_cmd(%w(activation-key update --organization org1 --name ak --auto-attach false ))
```
Comment (akofink) [inline test/functional/activation_key/update_test.rb]: Was this intentional?

Comment (sseelam2) [reply test/functional/activation_key/update_test.rb]: Oh no. Sorry. That's a mistake.

File: test/functional/activation_key/info_test.rb (hunk 1 above)
Comment (sseelam2) [reply test/functional/activation_key/info_test.rb]: ah alright. I'll make that change. Thanks.

File: test/functional/activation_key/info_test.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    params = ['--id=1']
+    ex = api_expects(:activation_keys, :show, "Activation key") do |par|
+      par['id'] == 1
+    end
+
+    json_file = File.join(File.dirname(__FILE__), 'data', 'activation_key.json')
+    ex.returns(JSON.parse(File.read(json_file)))
+
+    result = run_cmd(@cmd + params)
+    # rubocop:disable Style/WordArray
```
Comment (akofink) [inline test/functional/activation_key/info_test.rb]: This is not required and is causing the build to fail.

Comment (sseelam2) [reply test/functional/activation_key/info_test.rb]: alright. I have removed it.

**Pull request merged**